### DISCUSSION TITLE
Cleaning Storage

### DISCUSSION THREAD

**Discussion by BSSE23011** [community/community#66981] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hey Everyone!!!!!!!! I have a Macbook Pro late 13, 256 GB with a 8GB RAM. I don't know how it has completely got full and it is very frustrating as i can't download my programming softwares. The space that can be used is fluctuating from 20 MB to 500 MB. The storage distribution shows that other files is taking up a 80GB space but i can't find those files. I have searched every corner of storage but didn't find those 80GB files. Kindly provide a correct way to just get rid of that 80GB or maybe any free software that can help in clearing up that all.

**Comment by Akash1134** [comment]
Hi @BSSE23011 

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this topic. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
